vnext/Desktop/WebSocketModule.cpp

```diff
@@ -59,15 +64,15 @@ std::vector<facebook::xplat::module::CxxModule::Method> WebSocketModule::getMeth
           }
         }
 
-        this->GetWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0))->Connect(protocols, options);
+        this->GetOrCreateWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0))->Connect(protocols, options);
```

Comment:
>GetOrCreateWebSocket [](start = 14, length = 20)

It's a bit weird that if WebSocket is created, it has no owner. What is the pattern here?